stdlib/Std/src/Base.luna

```diff
@@ -974,11 +974,24 @@ class Map k v:
                     l' = l . delete key
                     Map._bin k v l' r . _balanceR
 
+class Scientific:
+    coefficient :: Int
+    exponent :: Int
+
+    def toText: case self.exponent >= 0 of
```

Comment:
Doesn't `if-then-else` work? This looks quite funny ;)

Reply:
You can't have a block as function argument.

Reply:
Oh, right. Btw. why not? Seems to fit fine within the whole suspended-computations idea.